docs/how-to/use-layers.md

```diff
@@ -0,0 +1,223 @@
+# How to use layers
+
+Managing multiple services across different environments becomes complex as systems scale. Pebble simplifies this with layered configurations, improving readability and maintainability.
+
+A base layer defines common settings (such as logging), while additional layers handle specific services or environment-specific overrides. This declarative approach, along with delegated layer management, allows for better cross-team collaboration and provides a clear view of each environment's configuration. For example, an operations team could manage base layers for logging, and service teams could manage layers for their services.
+
+(use_layers_pebble_layers)=
+## Pebble layers
+
+A layer is a configuration file that defines the desired state of the managed services.
+
+Layers are organized within a `layers/` subdirectory in the `$PEBBLE` directory. Their filenames are similar to `001-base-layer.yaml`, where the numerically prefixed filenames ensure a specific order of the layers, and the labels after the prefix uniquely identifies the layer. For example, `001-base-layer.yaml`, `002-override-layer.yaml`.
+
+A layer can define service properties (command, startup behavior, dependencies, environment variables, and more), health checks, and log targets. For example:
+
+```yaml
+services:
+  server:
+    override: replace
+    command: flask --app hello run
+    after:
+      - srv2
+    requires:
+      - srv2
+    environment:
+      PORT: 5000
+      DATABASE: dbserver.example.com
+  database:
+    override: replace
+    command: postgres -D /usr/local/pgsql/data
+    before:
+      - srv1
+checks:
```

Comment:
Here and below: let's add a blank line between top-level sections for readability.
```suggestion

checks:
```